### PR TITLE
ZFIN-8770: no adding ncbi genes manually

### PR DESCRIPTION
**Pull request by rtaylorzfin**

### CODE REVIEW

File: home/javascript/react/containers/MarkerEditDbLinks.js [hunk 1]
```diff
@@ -82,9 +82,10 @@ const MarkerEditDbLinks = ({markerId, group = 'other marker pages'}) => {
                         validate={value => value ? false : 'A database is required'}
                     >
                         <option value='' />
-                        {databases.value.map(database => (
-                            <option value={database.zdbID} key={database.zdbID}>{database.name}</option>
-                        ))}
+                        {databases.value
```
Comment (rtaylorzfin) [inline home/javascript/react/containers/MarkerEditDbLinks.js:85]: Look at the backend and check if we need to add some logic to the backend.

Comment (rtaylorzfin) [reply home/javascript/react/containers/MarkerEditDbLinks.js:85]: Better to handle this on the backend

**Comment by rtaylorzfin** [comment]
I tried a different approach with the latest commits.  I added "permissions" columns to the foreign_db_display_group_member table.  Then I marked the "can_add" permission as false for the row that puts NCBI Genes in the "other markers" display group.  This signifies that curators are not allowed to add NCBI Genes.

**Review by cmpich — APPROVED** [review]
Looks good.

**Pull request merged**